build/configs/rtl8730e/hello/defconfig

```diff
@@ -309,7 +310,7 @@ CONFIG_HW_RSA_ENC=y
 CONFIG_HW_SE_STORAGE=y
 # CONFIG_HW_AES_ENC is not set
 CONFIG_SE_SECURE_CONTEXT_SIZE=9216
-CONFIG_SECURITY_LEVEL=y
+# CONFIG_SECURITY_LEVEL is not set
```

Comment:
Let's revert this change in all defconfigs you modified.

Reply:
Done